Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationIdentification.cshtml.cs

```diff
@@ -110,6 +112,10 @@ public class OrganisationIdentificationModel(ISession session,
 
     public string? Identifier { get; set; }
 
+    public string? OrganisationName;
+
+    public FlashMessage NotificationBannerCompanyAlreadyRegistered { get { return new FlashMessage("An organisation with this registration number already exists. Change the registration number or <a class='govuk-notification-banner__link' href='/registration/" + Identifier + "/join-organisation'>request to join " + OrganisationName + ".</a>"); } }
```

Comment:
Use [string interpolation](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated) to build strings